spec/mongo/session_transaction_spec.rb

```diff
@@ -152,7 +152,7 @@ class SessionTransactionSpecError < StandardError; end
         it 'times out' do
           start = Mongo::Utils.monotonic_time
 
-          10.times do |i|
+          11.times do |i|
```

Comment:
I am wondering why increasing this.

Reply:
It's a long story but basically these expectations are ordered, so they're expecting exactly 11 calls to `monotonic_time`. I added a call to this method so it was throwing this test off.